src/components/tiles/table/use-content-change-handlers.ts

```diff
@@ -118,16 +119,38 @@ export const useContentChangeHandlers = ({
     getContent().removeCases(rowIds);
   }, [readOnly, getContent]);
 
-  const linkGeometryTile = useCallback((geomTileInfo: ITileLinkMetadata) => {
-    !readOnly && requestGeometryLinkToTable(getContent(), geomTileInfo.id);
-  }, [getContent, readOnly]);
+  const linkTile = useCallback((tileInfo: ITileLinkMetadata) => {
+    // !readOnly && requestGeometryLinkToTable(getContent(), geomTileInfo.id);
+    // The below makes the Geometry model's addLinkedTable action redundant.
+    // Should we instead add an addLinkedTable action to the Graph model,
+    // and then do something more similar to requestGeometryLinkToTable here?
+    const consumerTile = getTileContentById(getContent(), tileInfo.id);
```

Comment:
Ultimately, the Geometry tile should follow the pattern used here, rather than this code making accommodations for the Geometry tile.